app/src/components/main/modal/modalLayout/modalContent/modalContent.jsx

```diff
@@ -5,14 +5,14 @@ import styles from './modalContent.scss';
 
 const cx = classNames.bind(styles);
 
-export const ModalContent = ({ children, fullWidthContent }) => (
-  <div className={cx('modal-content', { 'full-width': fullWidthContent })}>{children}</div>
+export const ModalContent = ({ children, stretchedContent }) => (
```

Comment:
Now we don't need  stretchedContent prop. Remove it please.

Reply:
Fxied